subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaUpToDatePerformanceTest.groovy

```diff
@@ -21,11 +21,13 @@ import org.gradle.performance.AbstractCrossVersionPerformanceTest
 import org.gradle.performance.fixture.BuildExperimentListenerAdapter
 import org.gradle.performance.fixture.BuildExperimentSpec
 import org.gradle.test.fixtures.file.TestFile
+import spock.lang.Ignore
 import spock.lang.Unroll
 
 import static org.gradle.performance.generator.JavaTestProject.LARGE_JAVA_MULTI_PROJECT
 import static org.gradle.performance.generator.JavaTestProject.LARGE_MONOLITHIC_JAVA_PROJECT
 
+@Ignore("Rebase test as soon as merged to master")
```

Comment:
Just FYI you could push a nightly from your branch and rebaseline on top of that.

Reply:
Yeah, and afterwards I need to publish a nightly from `master` and rebaseline on that nightly. I really don't want to have a baseline from a branch on `master`.

Reply:
Not sure what the issue with that would be, since the code from this branch will be on master, so why not the baseline. But up to you anyway :)